W-15962448 login on install issue duke (#342)
mulesoft/docs-code-builder: modules/ROOT/pages/_partials/acb-start-desktop.adoc

+
For guidance, see xref:anypoint-code-builder::start-acb.adoc#change-clouds[Set the Desktop IDE to the EU or US Cloud].
. Follow the steps in xref:anypoint-code-builder::start-acb.adoc#login-ide[Log in to Anypoint Platform from the IDE].
+
NOTE::

> If an Anypoint Platform session is active in any of your browsers when you install Anypoint Extension Pack, the extension uses that session to log you in to Anypoint Platform from the IDE. You can log out and log in to the IDE at any time. 

Author: ```suggestion
If an Anypoint Platform session is active in any of your browsers when you install Anypoint Extension Pack, the extension pack uses that session to log you in to Anypoint Platform from the IDE. You can log out and log in to the IDE at any time. 
```

> If an Anypoint Platform session is active in any of your browsers when you install Anypoint Extension Pack, the extension pack uses that session to log you in to Anypoint Platform from the IDE. You can log out and log in to the IDE at any time. 

Author: ```suggestion
If an Anypoint Platform session is active in any of your browsers when you install Anypoint Extension Pack, the extension pack uses that session to log you in to Anypoint Platform from the IDE. You can xref:anypoint-code-builder::start-acb.adoc#logout-status-bar[log out] or  xref:anypoint-code-builder::start-acb.adoc#logout-status-bar[log in] to the IDE from the status bar. 
```

> If an Anypoint Platform session is active in any of your browsers when you install Anypoint Extension Pack, the extension pack uses that session to log you in to Anypoint Platform from the IDE. You can xref:anypoint-code-builder::start-acb.adoc#logout-status-bar[log out] or  xref:anypoint-code-builder::start-acb.adoc#logout-status-bar[log in] to the IDE from the status bar. 

Author: ```suggestion
If an Anypoint Platform session is active in any of your browsers when you install Anypoint Extension Pack, the extension pack uses that session to sign in to Anypoint Platform from the IDE. You can xref:anypoint-code-builder::start-acb.adoc#logout-status-bar[log out] or  xref:anypoint-code-builder::start-acb.adoc#logout-status-bar[sign in] from the status bar in the IDE. 
```
